### PR TITLE
MOBILE-729: Use `checking_for_updates` layout when finishing update too

### PR DESCRIPTION
**Pull request by NedelcuRadu**
Seems like there was a second identical layout used for "finishing_update" ;(

JIRA
https://almer-ar.atlassian.net/browse/MOBILE-729

### CODE REVIEW

**Review by oleksandr-hulak — APPROVED** [review]
Woow, I just saw how many different fragments there are, and layouts. Holy. Okay, let's do only what Alessandro drew, and see if it's enough or if they want to fix UI on other screens too.

**Comment by NedelcuRadu** [comment]
Yeah now it makes a bit more sense. I will make it on monday so during downloading the screen has `Download` and during install it's `Installing`

**Comment by amsurana** [comment]
@NedelcuRadu @oleksandr-hulak 

We should have NOT modified `master` branch... I added `force_ota` branch.

Did you guys knew about `force_ota` branch?

**Review by amsurana — CHANGES_REQUESTED** [review]
`master` shouldn't be modified. Use `force_ota` branch

Edit: My apologies.... The code from `force_ota` is merged in master because of AOSP compilation. So you guys are doing it right.

**Comment by NedelcuRadu** [comment]
This should be the final version, please review carefully as I can't run this project

File: MemfaultPackages/bort-ota/src/main/java/com/memfault/bort/ota/UpdateActivity.kt [hunk 1]
```diff
@@ -172,6 +174,9 @@ class CheckingForUpdatesFragment : Fragment() {
             progressBar.max = 100
             progressBar.progress = progress
         }
+        if (this::textView.isInitialized) {
+            textView.text = getString(R.string.downloading_format, progress)
```
Comment (oleksandr-hulak) [inline MemfaultPackages/bort-ota/src/main/java/com/memfault/bort/ota/UpdateActivity.kt:178]: From the logic above, we shoould also write "Downloading", if the progress is negative

**Review by oleksandr-hulak — CHANGES_REQUESTED** [review]

File: MemfaultPackages/bort-ota/src/main/java/com/memfault/bort/ota/UpdateActivity.kt (hunk 1 above)
Comment (NedelcuRadu) [reply MemfaultPackages/bort-ota/src/main/java/com/memfault/bort/ota/UpdateActivity.kt:178]: I don't get what you mean. Checking if the progress is negative before setting it? Why would the progress ever be negative

Comment (oleksandr-hulak) [reply MemfaultPackages/bort-ota/src/main/java/com/memfault/bort/ota/UpdateActivity.kt:178]: The progress can be negative in some situations, if we look 5 lines above: `progressBar.isIndeterminate = progress == -1`

**Review by oleksandr-hulak — APPROVED** [review]

**Pull request merged**